src/main/java/racing_car/step03/Race.java

```diff
@@ -0,0 +1,28 @@
+package racing_car.step03;
+
+import java.util.Random;
+
+public class Race {
+    private Random r = new Random();
+    private final int MIN_NUMBER = 4;
```

Comment:
지난번 피드백인 의미있는 이름을 가진 상수로 추출해보는것이 있었는데요, 이번에 잘 시도하신거같아요 👍
```suggestion
    private static final int MIN_NUMBER = 4;
```